pyproject.toml

```diff
@@ -26,11 +27,24 @@ Home = "https://archlinux.org"
 Documentation = "https://archinstall.readthedocs.io/"
 Source = "https://github.com/archlinux/archinstall"
 
+[project.optional-dependencies]
+doc = ["sphinx"]
+
 [project.scripts]
 archinstall = "archinstall:run_as_a_module"
 
-[project.optional-dependencies]
-doc = ["sphinx"]
+[tool.setuptools]
+packages = ["archinstall", "profiles", "examples"]
+
+[tool.setuptools.package-data]
```

Comment:
It's to avoid `setuptools` trying to auto-detect sub-packages etc. It would constantly detect the locales as a subpackage as `archinstall.locales` for instance. This should avoid that from happening. We can probably even remove `profiles` and `examples` but I wanted to be sure those got included in the packaging, even tho `package-data` should be responsible for that part.